scripts/jenkins/ardana/heat-ardana-std-min.yaml

```diff
@@ -0,0 +1,207 @@
+heat_template_version: 2016-10-14
```

Comment:
The file looks like a almost identical copy of the template we already have. is it possible to extract the duplicated parts somehow?

Reply:
dunno yet, wanted to do that after I understood what is is going to look like